docs/src/rules/sort-imports.md

```diff
@@ -239,18 +252,20 @@ import a from 'bar.js'
 
 :::
 
-Examples of **correct** code for this rule with the `{ "ignoreDeclarationSort": true }` option:
+Examples of **correct** code for this rule with the default `{ "ignoreDeclarationSort": false }` option:
 
 ::: correct
 
 ```js
-/*eslint sort-imports: ["error", { "ignoreDeclarationSort": true }]*/
-import a from 'foo.js'
-import b from 'bar.js'
+/*eslint sort-imports: ["error", { "ignoreDeclarationSort": false }]*/
+import a from 'bar.js'
+import b from 'foo.js'
```

Comment:
```suggestion
import a from 'bar.js';
import b from 'foo.js';
```

there are other examples that don't have semicolon can you please add semicolon to them too?

and also i think its better if we add an incorrect section for `{ "ignoreDeclarationSort": true }` with these examples that shows multiple members import are not allowed by this option:
```js
import {b, a, c} from 'foo.js';
```